### DISCUSSION TITLE
Add Valibot resolver

### DISCUSSION THREAD

**Discussion by blasdfaa** [mantinedev/mantine#4688] [Feature requests]
Add a resolver for `@mantine/form` to support the Valibot library.

A nice and small alternative to zod with similar api
https://github.com/fabian-hiller/valibot

**Comment by bogordesaincom** [comment]
how to implementation this?

**Comment by songkeys** [comment]
Hi. This has been supported. See: <https://mantine.dev/form/schema-validation/#valibot>
